src/Tmds.LinuxAsync/SocketsEventSource.cs

```diff
@@ -0,0 +1,13 @@
+using System.Diagnostics.Tracing;
+
+namespace Tmds.LinuxAsync
+{
+    [EventSource(Name = "SocketsEventSource")]
+    internal class SocketsEventSource : EventSource
+    {
+        [Event(1, Level = EventLevel.LogAlways)]
+        public void SocketEventsPerWait(int eventsCount) => WriteEvent(1, eventsCount);
```

Comment:
@adamsitnik I wonder about the added value of implementing this as an `EventCounter`. I guess the added value is mostly when you have some tooling that can work with `EventCounter`, but I don't think that applies for us. Does that make sense?

Reply:
Honestly, I had no idea about the existence of [EventCounter](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md) but after reading the docs it looks like a great fit for this particular event. The question is - is it supported by Event Pipe? cc @noahfalk

Reply:
Yes, it's supported by EventPipe. `EventCounter` is the x-plat `PerformanceCounter`.
I'll turn this into an `EventCounter` then.